lib/src/vIPT.cpp

```diff
@@ -31,6 +31,7 @@ namespace ev {
 vIPT::vIPT()
 {
     size_shared = Size(0, 0);
+    Q = cv::Mat(4, 4, CV_64FC1);
```

Comment:
I would allocate the Q matrix in the configure function with the other matrices for consistency.

Reply:
pushed new commit